src/nbl/video/device_capabilities/device_limits.json

```diff
@@ -135,7 +135,7 @@
                 {
                     "type": "uint32_t",
                     "name": "maxBoundDescriptorSets",
-                    "value": 4,
+                    "value": 8,
```

Comment:
I saw Metal only having 7 and roadmap only mandating 7, also Nabla is hardcoded to 4 so leave as 4